src/runtime/runtime_cortexm_hardfault_config_mimxrt1062.go

```diff
@@ -0,0 +1,27 @@
+// +build nxp,mimxrt1062
```

Comment:
Same as previous comment about adding to `device` package.

Reply:
Moved to new file `src/device/nxp/hardfault_mimxrt1062.go`